tutorpicasso/commands/enable_private_packages.py

```diff
@@ -0,0 +1,87 @@
+import os
+import subprocess
+from typing import Any, Dict
+
+import click
+from packaging.version import Version
+from tutor import config as tutor_config
+from tutor.__about__ import __version__ as tutor_version
+
+
+@click.command(name="enable-private-packages", help="Enable picasso private packages")
+def enable_private_packages() -> None:
```

Comment:
What do you think about having this caller invoke two additional private functions:

1. _enable_private_packages_before_quince
2. _enable_private_packages

In `_enable_private_packages_before_quince` would be the specific logic for installing private packages in < Quince and the other specifics for the newest versions. The caller will implement shared definitions between the two functions. This way, dropping or adding support for tutor versions might be more manageable. Let me know if you have any other suggestions to handle this. Thanks!

Reply:
Done